deployment/hasura/metadata/databases/tables/merlin/external_source.yaml

```diff
@@ -0,0 +1,62 @@
+table:
+  name: external_source
+  schema: merlin
+configuration:
+  custom_name: "external_source"
+array_relationships:
+  - name: external_events
+    using:
+      foreign_key_constraint_on:
+        columns:
+          - source_key
+          - derivation_group_name
+        table:
+          name: external_event
+          schema: merlin
+object_relationships:
+  - name: external_source_type
+    using:
+      foreign_key_constraint_on: source_type_name
```

Comment:
While currently it doesn't provide much of a benefit, as fields are added to `derivation_group` in the future, this could become useful. This relationship has now been added.